### PR TITLE
Source Greenhouse

### PR DESCRIPTION
**Pull request by eugene-kulak**
## What
Add new source Greenhouse

## How
*Describe the solution*

## Checklist
- [ ] *Create config API model*
- [ ] *Implement config persistence*
- [ ] *Add persistence tests*

## Recommended reading order
1. `test.java`
1. `component.ts`
1. the rest

### CODE REVIEW

**Comment by eugene-kulak** [comment]
based on #1068

File: airbyte-integrations/connectors/source-greenhouse/Dockerfile.test [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+ENV AIRBYTE_TEST_MODULE="integration_tests"
+ENV AIRBYTE_TEST_PATH="SourceGreenhouseStandardTest"
+
+LABEL io.airbyte.version=0.1.0
+LABEL io.airbyte.name=airbyte/source-greenhouse-standard-test
+
+WORKDIR /airbyte/integration_code
+COPY source_greenhouse source_greenhouse
+COPY $CODE_PATH $CODE_PATH
+COPY sample_files/*.json $CODE_PATH
```
Comment (sherifnada) [inline airbyte-integrations/connectors/source-greenhouse/Dockerfile.test:15]: needed by docker if there are multiple files being copied

```suggestion
COPY sample_files/*.json $CODE_PATH/
```

**Review by sherifnada — COMMENTED** [review]
This is pretty close to shipping, the main two things remaining are: 
1. merge master to pick up the changes in this PR #1145 and resolve any conflicts
2. run `./tools/integrations/manage.sh build airbyte-integrations/connectors/source-greenhouse` from the repo root and verify the tests are passing

**Comment by eugene-kulak** [comment]
`./tools/integrations/manage.sh build airbyte-integrations/connectors/source-greenhouse`
```
Test run finished after 63178 ms
[         2 containers found      ]
[         0 containers skipped    ]
[         4 containers started    ]
[         0 containers aborted    ]
[         4 containers successful ]
[         0 containers failed     ]
[         5 tests found           ]
[         0 tests skipped         ]
[        10 tests started         ]
[         0 tests aborted         ]
[        10 tests successful      ]
[         0 tests failed          ]


Deprecated Gradle features were used in this build, making it incompatible with Gradle 7.0.
Use '--warning-mode all' to show the individual deprecation warnings.
See https://docs.gradle.org/6.7.1/userguide/command_line_interface.html#sec:command_line_warnings

BUILD SUCCESSFUL in 2m 32s
86 actionable tasks: 37 executed, 49 up-to-date
```

**Review by sherifnada — APPROVED** [review]
LGTM -- will merge in the morning

**Pull request merged**